<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SI_UnitSetWid</class>
 <widget class="QWidget" name="SI_UnitSetWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>33</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="minBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="maxBtn">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="maxSpinBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="minSpinBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="decimals">
    <number>1</number>
   </property>
   <property name="maximum">
    <double>999.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
